README.md

```diff
@@ -2,25 +2,27 @@
 
 ## Learning Goals
 
-- Recognize how multiple `class`es can be used together
+- Recognize how multiple `class`es can share data
+- Recognize how multiple `class`es can share behaviors
+- Recognize how `class`es working together develop dependencies
 
 ## Introduction
 
 Using Object Oriented JavaScript, we can encapsulate closely related data and
-behavior. Using `class`es, properties, methods, getters and setters, we can
-finely tune how our data is accessed and modified. And since everything is
-wrapped up, together, no matter where an `class` instance is needed, it behaves
-in a way we've defined. A single cell in a soup of code.
+behaviors. Using `class`es, properties, methods, getters and setters, we are
+able to finely tune how our data is accessed and modified. And since everything
+is wrapped up together, no matter where an `class` instance is needed, it
+behaves in the way we've defined.
 
 With that foundation, we're now going to look at what happens when we write
 multiple, related `class`es. Object Orientation is particularly effective
-when it comes to representing systems of inter-related things.
+when it comes to representing systems of interrelated things.
 
 In this lesson, we're going to look at some of the ways in which
-`class`es can interact, and the benefits that `class` relationships can
+`class`es can interact and the benefits that `class` relationships can
 provide us.
 
-## Multiple Class Objects Reduce
+## Sharing Data between Multiple Classes
```

Comment:
Should this be revised to use the template for reiterating the learning goals and headings?